docs/source/technical_tutorials/access/aws_access.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>Link to GDAL OGR driver list. <a href="https://gdal.org/en/stable/drivers/vector/index.html" target="_blank">https://gdal.org/en/stable/drivers/vector/index.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>